rmc-docs/src/tutorial-loops-unwinding.md

```diff
@@ -0,0 +1,129 @@
+# Loops, unwinding, and bounds
+
+Consider code like this:
+
+```rust
+{{#include tutorial/loops-unwinding/src/lib.rs:code}}
+```
+
+And a proof harness like this:
+
+```rust
+{{#include tutorial/loops-unwinding/src/lib.rs:rmc}}
+```
+
+When we run RMC on this, we run into an unfortunate result: non-termination.
```

Comment:
It would be useful to clarify the source of non-termination, which is that it unrolls the loop an unbounded number of times.